### ISSUE TITLE
F-Droid

### ISSUE THREAD

**Issue by vdbhb59** [AnySoftKeyboard/AnySoftKeyboard#2334]
The language lists on F-Droid have not been updated for a long time now. Can this be updated please?
Also, on F-Droid, the ASK search shows odd names, like "Catalan For ASK", "Indonesian for ASK", and not the ASK main application itself. Not sure of it is F-Droid metadata issue or what, but of shows different things to many. Kindly fix that too.
vdbhb59

**Comment by vdbhb59** [comment]
Anyone? @menny prolly?

**Comment by Porrumentzio** [comment]
We're having the same issue for time ago. Some people see "Euskarazko hizkuntza paketea", others "Kachin For ASK", others "Italian For ASK", instead of simply AnySoftKeyboard (main app).

Also there are problems with the czech package, as it takes other names.

**Comment by vdbhb59** [comment]
Developers, kindly assist, as I see you are updating language packs and the main application (beta as well) on playstore, but you are seemingly not updating anything nor fixing on F-Droid.
Please update, as I am sure you too respect people wishing privacy.

**Comment by l3u** [comment]
I also ran into this (as emerged in #2352) and would also greatly appreciate being able to install the same ASK version from F-Droid as from the Play Store.

**Comment by Porrumentzio** [comment]
Again on fresh installed eOS with basque locales.

1) The first `Euskarazko hizkuntza paketea` (what means "**Basque** language pack"), with the **armenian** flag, is the [**main** app](https://f-droid.org/packages/com.menny.android.anysoftkeyboard).
2) The second `Euskarazko hizkuntza paketea`, with the **french flag**, is the [**Czech** language package](https://f-droid.org/packages/org.herrlado.ask.languagepack.czech).


This is really, really confusing!!

![irudia](https://user-images.githubusercontent.com/61913683/87244648-dd6ac580-c43e-11ea-80e5-6eb3ad5062a3.png)

**Comment by landry314** [comment]
on f-droid it says "Turkish for Anysoftkeyboard" instead of "Anysoftkeyboard" for the main app. Weird bug.

**Comment by vdbhb59** [comment]
Is any developer even going to work on this, ever? F-Droid is not getting any update, and you are concentrating only on beta and that too on f**kstore.

**Comment by Porrumentzio** [comment]
It's not only F-Droid, but also /e/ Apps, which is a repository for /e/ Android System. Seems that all except Google Play does not work well.

**Comment by vdbhb59** [comment]
Still no response from the developer. At least give a holding response mate..

**Comment by Porrumentzio** [comment]
Another comment related to this: https://github.com/AnySoftKeyboard/AnySoftKeyboard/issues/2351#issuecomment-679172879

**Comment by vdbhb59** [comment]
@Porrumentzio
Yeah, I saw all. Not sure what the trouble is? F-Droid version is so old.

**Comment by Porrumentzio** [comment]
They are not updating F-Droid versions neither taking care about their metadata, seemly

**Comment by vdbhb59** [comment]
@Porrumentzio true. Even not caring. I will see if I can fork and update for F-Droid or a FOSS version, but not sure if I can get time for it..

**Comment by vdbhb59** [comment]
@menny
Are you wishing to update the F-Droid version anymore? Else I will close out this fruitless topic, which is just causing disappointment to all knowing that more and more open source stuff is migrating to stupidstore and becoming privacy mess.

**Comment by Porrumentzio** [comment]
Maybe @nicoursi could help 🤷

**Comment by felschr** [comment]
Can we at least have the F-Droid listing removed?
I've been on an old version for over half a year thinking ASK isn't getting any new stable releases for whatever reason.

**Comment by landry314** [comment]
Why not just push the betas to F-Droid? That would get you more beta testers. F-Droid people are more experimental anyway and know how to use github.

**Comment by bonswouar** [comment]
> Can we at least have the F-Droid listing removed or add a warning?
> I've been on an old version for over half a year thinking ASK isn't getting any new stable releases for whatever reason.

Same here, except now there has been one on GooglePlay since then...

I really don't see the point of letting this old version on F-Droid. I know the developer is kinda alone but that's not really something development-related ! If we'd understand why it's stuck I'm sure some of us would be glad to help.

(Although this not the first time I notice updates coming much later on F-Droid, what is the problem with this store ?!)

**Comment by vdbhb59** [comment]
Actually it is not the developer being alone. Nor is it F-Droid's problem. The ASK team/author has never responded here, and never seems to bother with open source. Just check the ASK Beta's. If you can push to fooglestore, you surely can push one at least to FD as well. How does it differ? I am on FD, and know their requirements for years now. It is the best one. But ASK I think is into monetisation from PS or some kind of binding that he does not care for opensource, though that is where he started with ASK in the first place. This is with every other open source stuff. I do not say trying for money is not good, but then do not call yourself an advocate of open source and community, when all you care for is fooglestore.
